CleanWeather/Scenes/CityListMVVM/CityListCellManagerMVVM.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CityListCellManagerMVVM.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 11/01/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+protocol CityListCellManagerMVVMDelegate: class {
+    func didSelectCityCell(city: CityWeather)
+}
+
+final class CityListCellManagerMVVM: NSObject {
```

Comment:
Do you have to use nsobject?

Reply:
If I don't use it here then I get error
`Type 'CityListCellManagerMVVM' does not conform to protocol 'NSObjectProtocol'`
And to fix that I have to import 13 methods:
```
func isEqual(_ object: Any?) -> Bool {
        <#code#>
    }
    
    var hash: Int {
        <#code#>
    }
    
    var superclass: AnyClass? {
        <#code#>
    }

    func `self`() -> Self {
        <#code#>
    }

    func perform(_ aSelector: Selector!) -> Unmanaged<AnyObject>! {
        <#code#>
    }

    func perform(_ aSelector: Selector!, with object: Any!) -> Unmanaged<AnyObject>! {
        <#code#>
    }

    func perform(_ aSelector: Selector!, with object1: Any!, with object2: Any!) -> Unmanaged<AnyObject>! {
        <#code#>
    }

    func isProxy() -> Bool {
        <#code#>
    }

    func isKind(of aClass: AnyClass) -> Bool {
        <#code#>
    }

    func isMember(of aClass: AnyClass) -> Bool {
        <#code#>
    }

    func conforms(to aProtocol: Protocol) -> Bool {
        <#code#>
    }

    func responds(to aSelector: Selector!) -> Bool {
        <#code#>
    }

    var description: String {
        <#code#>
    }
```